src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,79 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V> existedNode = getNode(key);
+        if (existedNode != null) {
+            existedNode.value = value;
+            return;
+        }
+        int index = getIndex(key);
+        table[index] = new Node<>(key, value, table[index]);
+        size++;
+        if ((float) size / table.length >= LOAD_FACTOR) {
+            resize();
+        }
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        return getNode(key) == null ? null : getNode(key).value;
```

Comment:
your current solution has drawback, you calculate `getNode()` twice, should be extracted in local var.
```suggestion
var node = getNode(key);
        return node == null ? null : node .value;
```